setup.py

```diff
@@ -12,7 +12,7 @@
 INSTALL_REQUIRES = [
     "Django>=4.0",
     "cryptojwt>=1.8.2",
-    "pydantic>=1.8.2",
+    "pydantic==1.8.2",
```

Comment:
```suggestion
    "pydantic>=1.8.2,<=1.9.0",
```

I'd avoid `==` since it produces issues with misalignements with future version of other packaegs


@rglauco what about movint to https://github.com/pydantic/pydantic/releases/tag/v2.0?